src/Columns/IColumn.h

```diff
@@ -272,12 +272,16 @@ class IColumn : public COW<IColumn>
     /// Return pointer to the end of the serialization data.
     virtual char * serializeValueIntoMemory(size_t /* n */, char * /* memory */) const;
 
+    virtual void batchSerializeValueIntoMemory(std::vector<char *> & /* memories */) const;
```

Comment:
Looks good. As long as the vector has to be already in the right size, let's add `const`